src/themes/corteza-base/custom.scss

```diff
@@ -67,3 +67,72 @@ html {
   background-position: center;
   background-repeat: no-repeat;
 }
+
+.ProseMirror.ProseMirror-focused {
+  outline: none;
+}
+
+// Rich text content styling
+.rt-content {
+  p {
```

Comment:
can you use p-0 and m-0?

Reply:
He... I actually can use those (since I've already had to re-define paragraphs for alignment). But... I don't want to clutter our document model with such things.